masschallenge/django-accelerator
# [AC-6538]: Delete old mentor directory profile

[AC-6538](https://masschallenge.atlassian.net/browse/AC-6538)

Changes introduced:
- The old mentor code was deleted
- The link reference was updated to use the new community directory

How to test:
- Ensure all the test passes
- mentor profile code are deleted
Sibling PR: https://github.com/masschallenge/accelerate/pull/3144
